chainer/functions/normalization/batch_normalization.py

```diff
@@ -426,12 +426,18 @@ def forward(self, inputs):
 
         self._impl = self._impl_selector(self, inputs)
 
+        axis = self.axis
```

Comment:
Let's use different name to avoid confusion.
```suggestion
        raw_axis = self.axis
```